src/domain/membership/graphql/MembershipInformation.graphql

```diff
@@ -1,7 +1,7 @@
 query MembershipInformation {
     myYouthProfile {
+        id
         profile {
-            id
```

Comment:
Seems like this `id` is used somewhere in the code.

Reply:
It is indeed. I thought I double checked everything, after taking another look it was used in the QRcode that points to admin. Added it back.